__docs__/wonder-blocks-button/button.stories.tsx

```diff
@@ -539,6 +481,11 @@ Size.parameters = {
             story: "Buttons have a size that's either `medium` (default), `small`, or `large`.",
         },
     },
+    chromatic: {
+        // We already have screenshots of other stories that cover more of
+        // the button states
+        disableSnapshot: true,
+    },
 };
 
 export const Spinner: StoryComponentType = () => (
```

Comment:
Would it be helpful to include the spinner state in All Variants as well?

Reply:
I thought about that one, but I decided to keep it separately as it could introduce flakyness to that snapshot. I'm mentioning that because this variant includes animation and I've seen that Chromatic sometimes takes screenshots at different times.

That said, maybe we could add it and see how it goes? wdyt?

Reply:
Good point! I was thinking it could be helpful to see it across the different themes/states/variants etc, though I understand snapshots for animated things can be flaky! I'm okay leaving it as a separate story!